### PR TITLE
Adapt ext point description for onboarding image

### PR DESCRIPTION
**Pull request by marcushoepfner**
Size shall be 250x250.

Issue: https://github.com/eclipse-platform/eclipse.platform.ui/issues/702

### CODE REVIEW

File: bundles/org.eclipse.ui/schema/perspectiveExtensions.exsd
Comment (BeckerWdf) [inline bundles/org.eclipse.ui/schema/perspectiveExtensions.exsd]:
```suggestion
Plus a second image for high resolution with a size of 500 x 500 px and a name like [image_name]@2x.[image_type].
```

**Review by BeckerWdf — CHANGES_REQUESTED** [review]

**Review by BeckerWdf — APPROVED** [review]

**Comment by iloveeclipse** [comment]
@marcushoepfner : please **do not ignore** "block merge commit" checks!
image

@laeubi : this is what I mean "people learn to ignore errors".

**Comment by laeubi** [comment]
@iloveeclipse you are **project lead** on platform, you can ask the infrateam to [enable branch protection rules](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule) that forbid merging if certain checks fail. 
So please don't blame the checks for being ignored by people, maybe better people should learn to not merge things blindly ...

**Comment by iloveeclipse** [comment]
> [enable branch protection rules](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule) that forbid merging if certain checks fail.

Actually I see someone enabled merge commits for platform UI. I've disabled that now. I will also check other repos now, they actually should be all with disabled merge.

**Comment by iloveeclipse** [comment]
> So please don't blame the checks for being ignored by people, maybe better people should learn to not merge things blindly ...

I will continue to blame the **failing** checks added without anyone committed to work on fixing the fails.
This causes **constant** pain for everyone and people don't understand if the failures are related to their changes or not.

**Comment by merks** [comment]
Probably merge was enabled to support merging in another repo and then not disabled again afterwards...

**Comment by laeubi** [comment]
> Actually I see someone enabled merge commits for platform UI. I've disabled that now. 

Requiring a sucessfull check is nothing that can be enabled by the "normal" people, for example currently one can even merge if all check fail.

> This causes constant pain for everyone and people don't understand if the failures are related to their changes or not.

No pain no gain ... I recently heard we have stabilization weeks where people only working on stabilize things?!? So if **everyone** that  feels the pain would help to fix a single test ... but it seems easier to complain and ignore.

**Comment by BeckerWdf** [comment]
> I will continue to blame the **failing** checks added without anyone committed to work on fixing the fails. This causes **constant** pain for everyone and people don't understand if the failures are related to their changes or not.

Yes this really is an issue.

**Comment by laeubi** [comment]
> > I will continue to blame the **failing** checks added without anyone committed to work on fixing the fails. This causes **constant** pain for everyone and people don't understand if the failures are related to their changes or not.
> 
> Yes this really is an issue.

The start fixing the issue or be more careful when merging. You ignored a check **that has never failed before** so please don't blame other checks for ignoring ones that work exactly as intended.

**Comment by merks** [comment]
It would be nice to be a bit nicer.  Yes, failing tests cause us to eventually ignore failing tests, and that is to our own peril.  Yes, when I come and fix one little problem, and I only have time for fixing that one little problem, I will tend to ignore unrelated failing tests. Yes, we should all feel responsible for fixing failing tests, even those that are not our own.   Who is at fault is not really so important.

**Comment by marcushoepfner** [comment]
> @marcushoepfner : please **do not ignore** "block merge commit" checks! image
> 
> @laeubi : this is what I mean "people learn to ignore errors".

Sorry for that. Broken window effect I guess. I just threw another stone.
I rewrote history now I think. Can this be fixed?

I now know the reason for that. In the PR I get an "update branch" button. The default is merge. The button can be changed to rebase ("drop down button"). But one has to know that.

**Comment by laeubi** [comment]
@merks a completely unrelated check (NOT regarding test failures) was ignored, and I assume it the **responsibility of the committer** to check if the test failure is really unrelated and not **ignore any failing check**.

Also please note that **it was explicitly requested** that failing test **should also fail the builds** here (before only the Junit check has failed but not the build):
- https://github.com/eclipse-platform/eclipse.platform.swt/issues/594

beside that we have had in the past enough examples where a regression was only discovered at the ibuilds (or even first at the user tests) because we before has only checked the **Linux** builds! So yes one can say before there was a happy time because we simply **don't see the issues**.

Also **there are people** that are working on fixing the test so I find it really unfair and not "friendly" for those that really try to improve things here that they have **not fixed all issue** for us and instead we are "annoyed"...

so sorry for being "not so nice" that now everyone uses some failing tests as an excuse to ignore common development practice and just merge as they find useful.

**Comment by merks** [comment]
@laeubi  

You are of course correct.  My "it's nice to be nice" comment was not direct only and specifically to you.  Given the pervasiveness and relevance of your contributions to make things work better and better, I think you most definitely are entitled to be treated nicely.   🏅  What goes around, comes around, so its just nicer to be nicer. 👼

**Comment by iloveeclipse** [comment]
> Broken window effect I guess. I just threw another stone.

Exact what I mean. Once the people are "used" to see errors **in every PR**, they start to ignore them.

> I rewrote history now I think. 

No, you didn't 

> Can this be fixed?

Nothing to fix now. Please just next time avoid merging if **anything** looks suspicious.

**Comment by laeubi** [comment]
> Once the people are "used" to see errors in every PR, they start to ignore them.

Please don't generalize that, I don't do that and other don't do that as well. And the check in question has not failed on _every_ PR, it has only failed on _this_ PR.

But nice to know, that next time the **freeze check** fails on **every** PR I can assume it can be ignored!

**Comment by laeubi** [comment]
> Given the pervasiveness and relevance of your contributions to make things work better and better

 there are also others that have done a great job recently to stabilize test:
- https://github.com/eclipse-platform/eclipse.platform/pulls?q=is%3Apr+author%3AHeikoKlare

Also @HannesWell has fixed several things on windows already
- https://github.com/eclipse-pde/eclipse.pde/pulls?q=is%3Apr+author%3AHannesWell

and I think it was you to being very eager and finally fixing the "661 test failures" at PDE ... so claiming no one is "dedicated" is really not true and I think that at least as valuable (if even not more valuable) than new features and beeing "annoyed".

> What goes around, comes around, so its just nicer to be nicer.

:+1:  I just take the opportunity here:
- https://github.com/eclipse-platform/eclipse.platform/issues/408

**Comment by iloveeclipse** [comment]
> Please don't generalize that, 

**I** do not generalize, is is a [well known theory](https://en.wikipedia.org/wiki/Broken_windows_theory). 
 
> But nice to know, that next time the **freeze check** fails on **every** PR I can assume it can be ignored!

This is not funny anymore. I would revert every commit that intentionally ignored freeze check.

**Pull request merged**
